.travis.yml

```diff
@@ -67,11 +60,22 @@ matrix:
     - python -m pip install -r cirq/contrib/contrib-requirements.txt
     - python -m pip install -r dev_tools/conf/pip-list-dev-tools.txt
     script: check/pytest --benchmark-skip --actually-quiet
-
   - os: linux
     env: NAME=doctest
-    python: "3.6"
+    python: '3.6'
     install:
-      - pip install -r requirements.txt
-      - pip install -r cirq/contrib/contrib-requirements.txt
+    - pip install -r requirements.txt
+    - pip install -r cirq/contrib/contrib-requirements.txt
     script: check/doctest -q
+
+before_deploy:
+  - source dev_tools/packaging/set-dev-version.sh
+
+deploy:
+  - provider: pypi
+    user: quantumlibtest
+    on:
+      condition: "$CIRQ_DEV_VERSION"
+    password:
+      secure: mgMHPX1AI7TnZoA3VqK8SWhcldbgoIfOysgI1X60lXbkePsYAjVO1NlgIaATAKpVHLYPlKlrdreJiSrZL+k6ACaTRzV5TN2Y8/vZSwCcDlloEpAj9CProC2DJFJeunL1uqN5kSL4hOSOi9/JlIdfUM7GoiZFUAol/SzQYtDunWozUvVjh8JGiqapjJ46iH8kpeOhgCYD4s7GdQ4L5gwSAlQzc5iu2JGpcRP/cPzB8aKL0Fgxnj+F0ajxgQFda9UNq7eyemRMtqLvaKlWDxeYiGEF6EazyHkN9Yz3r519j/nawLlESiY7kQ1/u0ZMxYfopN+M578uCT99yVRQeSFe7KeF9D5eanuDs1m1ZamkTgW+8Yw4ygZGvHcjkMbks14mCSHOem5Lyxo0DBsxEd4Q+adgBiF33T280qzGQK2IS4mxSk28v8dz9dTLM+0rcBSJkmJ6X61ma4Hi6Pb7pfSOulcudOd/sT3HWBtF4F3UF+XT9WR3SM9CgaEOFlbfPtQo0F/hiAvCn31fOh/Pa4uTisK9flf9MGN75wE8fEKY9KrOCe1eANIQE805I4y8jTsd3fblXhbgGjIvbdRSh1VuHQMS78JVJm7Te+2bQBBRwVrkjlZmuihfByCUBCgf+GVftPO/XjoEQNoSWp23BCOTGeHVmKcR0UfK548lg3bvYsY=
+    server: https://test.pypi.org/simple/
```

Comment:
Add `distributions: 'bdist_wheel'` to match `produce-package.sh` (Travis default is sdist).  https://docs.travis-ci.com/user/deployment/pypi/#uploading-different-distributions